packages/dev/core/src/Shaders/ShadersInclude/instancesDeclaration.fx

```diff
@@ -24,7 +24,7 @@
 	    uniform mat4 world;
     #endif
 #if defined(VELOCITY) || defined(PREPASS_VELOCITY) ||              \
-                defined(PREPASS_VELOCITY_LINEAR)
+                defined(PREPASS_VELOCITY_LINEAR) || defined(VELOCITY_LINEAR)

```

Comment:
Same (other occurrences in other files).

Reply:
I think I've got them all.